docs/executor/balsam_executor.rst

```diff
@@ -1,17 +1,35 @@
-Balsam MPI Executor
-===================
+Balsam Executors
+================
+
+Balsam 2 Executor
+-----------------
+
+.. automodule:: new_balsam_executor
+  :no-undoc-members:
+
+.. autoclass:: NewBalsamExecutor
+  :show-inheritance:
+  :members: __init__, register_app, submit_allocation, revoke_allocation, submit
+
+.. autoclass:: NewBalsamTask
+  :show-inheritance:
+  :member-order: bysource
+  :members: poll, wait, kill
+
+Balsam 1 MPI Executor
```

Comment:
Is there a reason "MPI" is here and not above?

Reply:
Our older balsam executor implementation only ever used MPI I believe, but the new one can also do serial jobs. It also doesn't inherit from the mpi executor